package_sync_helpers/tools.py

```diff
@@ -20,7 +20,7 @@ def get_psync_settings():
         "list_backup_path": s.get("list_backup_path", ""),
         "zip_backup_path": s.get("zip_backup_path", ""),
         "folder_backup_path": s.get("folder_backup_path", ""),
-        "ignore_files": s.get("ignore_files", []) + ["PackageSync.sublime-settings", "PackageSync.last-run"],
+        "ignore_files": s.get("ignore_files", []),
```

Comment:
Use the flag to define if PackageSync settings are to be synced.
Still, It is not advisable to use this feature. Would think about the repercussions of the same before merging.